web/packages/teleterm/src/ui/DocumentGatewayApp/AppGateway.tsx

```diff
@@ -256,35 +257,49 @@ export function AppGateway(props: {
   );
 }
 
-const LabelWithAttemptStatus = (props: {
-  text: string;
-  attempt: Attempt<unknown>;
-}) => (
-  <Flex
-    alignItems="center"
-    justifyContent="space-between"
+const LabelWithAttemptStatus = (
+  props: PropsWithChildren<{
+    text: string;
+    attempt: Attempt<unknown>;
```

Comment:
Can't we scope down attempt type to be one of `changeLocalPortAttempt` or `changeTargetPortAttempt`?

Reply:
Yeah, that's exactly why, I didn't bother writing out the type since attempt data is not used. One fewer place to change if we ever need to update it for some reason.

Reply:
@ravicious For the record, I think that a good alternative in such cases would be to simply pass status alone.

Reply:
True, that's an option too. I didn't think of it because whenever I work with `Attempt` my mental model is that of languages where the status would be an actual distinct member of a sum type, e.g. [RemoteData in Elm](https://package.elm-lang.org/packages/krisajenkins/remotedata/latest/RemoteData#RemoteData), rather than just a field on an object that you can detach and pass in isolation.